### DISCUSSION TITLE
Discussion on Metrics and Monitoring

### DISCUSSION THREAD

**Discussion by bjwhite-fnal** [rucio/rucio#6593] [Ideas]
I'd like to start a longer-running thread here, to be used for floating improvements to the Prometheus metrics, and how we monitor Rucio operations.

For example, we might be able to add more information to the metrics exported by Automatix by adding information about which RSE the operation was performed on. https://github.com/rucio/rucio/blob/bed3f050bc6ee12dc4e6a92637a628604b265725/lib/rucio/daemons/automatix/automatix.py#L233

The Reaper already does similar.
https://github.com/rucio/rucio/blob/bed3f050bc6ee12dc4e6a92637a628604b265725/lib/rucio/daemons/reaper/reaper.py#L173

If anyone has similar thoughts, this is the place for them!

**Comment by bari12** [comment]
I think that's a good idea, is there a list of metrics you have identified where this would be most useful (Other than in automatix?)
